source/clear-linux/guides/maintenance/swupd-guide.rst

```diff
@@ -0,0 +1,160 @@
+.. _swupd-guide:
+
+Use swupd
+#########
+
+*swupd* manages the software update capability of |CLOSIA|. It can check for
+valid system updates and, if found, download and install them. It can also
+perform verification of the system software. 
+
+|CL| uses :ref:`bundles-about<bundles>` as the base abstraction for
+installing functionality on top of the core operating system. Use the `swupd`
+tool to install and remove bundles.
+
+.. contents:: 
+   :local:
+   :depth: 2
+
+.. _man_swupd:
+
+For a full listing of commands and options please see the man page in |CL|
+
+.. code-block:: bash
+
+   man swupd
+
+OS Update and Verification
+**************************
+
+|CL| is designed to promote a regular update cadence. `swupd` helps to
+make sure that process is simple and secure.
+
+OS Info
+=======
+
+Current OS version and update server info:
+
+.. code-block:: bash
+
+   swupd info
+
+.. code-block:: console
+   Installed version: 23330
+   Version URL:       https://download.clearlinux.org/update/
+   Content URL:       https://cdn.download.clearlinux.org/update/
+
+Disable Automatic Updates
+=========================
+
+|CL| updates are automatic by default but can be toggled to occur only 
+on demand. Disable automatic updates:
+
+.. code-block:: bash
+
+   sudo swupd autoupdate --disable
+
+To re-enable automatic updates use the `--enable` flag.
```

Comment:
I'd include both 'enable' and 'disable' examples here--as shown originally in `update.rst`. 
(`--disable` option is an overlap with `update.rst`--which is another reason for combining docs.)